utils/eventtracking.js

```diff
@@ -109,7 +122,58 @@ const getEventTrackingConsent = async () => {
     }
 };
 
-const askForConsentIfNeeded = async (options) => {
+const getIdTrackingConsent = async () => {
+    const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+    });
+    try {
+        for (let attempts = 0; attempts < 10; attempts++) {
+            const answer = await new Promise((resolve) => {
+                rl.question(
+                    chalk`We would like to help with your development journey with NEAR.` +
+                        chalk` We will ask you to share your account Id while using command. ` +
```

Comment:
nit: I think we usually spell it as `account ID`, e.g. https://docs.near.org/docs/concepts/account#account-id-rules

Reply:
I think we also need to add a text to explain to user that this data is already generally on chain. Something like:

`Note that your account ID and all associated on-chain transactions are already being recorded on public blockchain.`

Maybe @kcole16 @corwinharrell can help with text here. Basically we should try to avoid surprises.

Reply:
I added